File: app-java/src/main/java/ir/map/servicesdk/sample/AppController.java
package ir.map.servicesdk.sample;

import android.app.Application;

import ir.map.servicesdk.MapirService;

public class AppController extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        //TODO Please add your API_KEY
        MapirService.init(getBaseContext(), getString(R.string.api_key));
    }
}

File: app-kotlin/src/main/java/ir/map/servicesdk/app_kotlin/AppController.kt
package ir.map.servicesdk.app_kotlin

import android.app.Application
import ir.map.servicesdk.MapirService

class AppController : Application() {

    override fun onCreate() {
        super.onCreate()

        //TODO Please add your API_KEY
        MapirService.init(baseContext, getString(R.string.api_key))
    }
}